1-exercises/F-for-of-loop/exercise.js

```diff
@@ -4,13 +4,24 @@
 
 // TODO Use a for-of loop to output each of the tube stations below.
 let tubeStations = [
-    "Aldgate",
-    "Baker Street",
-    "Picadilly Circus",
-    "Oxford Street",
-    "Tottenham Court Road"
+  "Aldgate",
+  "Baker Street",
+  "Picadilly Circus",
+  "Oxford Street",
+  "Tottenham Court Road",
```

Comment:
Yes, good use of trailing comma in array elements. This new JS feature allow us copying / shuffling array elements with ease and no need to remove the last comma all the time.